### ISSUE TITLE
Fields for create quarkus maven does not have keyboard focus

### ISSUE THREAD

**Issue by maxandersen** [redhat-developer/vscode-quarkus#73]
When starting Quarkus create maven project I seem to almost always have to use the mouse or tab to get the field to have focus. 

Expectation: not having to use the mouse nor tab around fields to go through the flow.

**Comment by xorye** [comment]
I'm having trouble replicating this issue.

When I call the command to create the project, the field is always highlighted for me. Because of this, I can just go ahead and start typing. After I'm done, I can hit Enter and the next field is also highlighted:
![Creating a new project](https://raw.githubusercontent.com/xorye/gifs/master/new_issues/project_gen.gif?token=AE3CR5I5WGOPQKYIPIAFMY25RDUSW)
Although, I have to use tab to use the back button without the mouse.

Are the fields not highlighted on your end?

**Comment by maxandersen** [comment]
I can't reproduce the exact issue  now - just remember that the field did not have focus.

What I'm seeing now is that the field does have focus (text is highlighted) but when pressing Enter on  project name it doesn't do anything.

I have to go edit it and then it highlights and says "Invalid artifactId"  for "MyQuarkusProject" (the default) which seem to be because it want it to have all lower case...which isn't invalid imo. I'll open separate issue on that.

**Comment by xorye** [comment]
Reopening because I came across this issue on a Windows machine

**Comment by xorye** [comment]
I've ran into this issue when I open the wizard when I have a webview open (vscode-quarkus welcome page, java extension pack overview, gitlens welcome page etc.)

This issue is likely related to this issue: https://github.com/microsoft/vscode/issues/76316
